easybuild/easyconfigs/g/GPAW/GPAW-1.3.0-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '1.3.0'
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+ method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or
+ atom-centered basis-functions."""
+
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://pypi.python.org/packages/11/59/bfb6102911f9c0c35bfd0c1f3f51e0f4c31523b9f700674d18e4f60d7081/']
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = [
+    'GPAW-1.3.0-dynamic_linking.patch',  # removes LIBPL from -L as linker only finds static library in this directory
+    'GPAW-1.3.0-mkl.patch',
+]
+
+python = 'Python'
+pythonver = '2.7.14'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % (python, pythonver)
```

Comment:
You don't actually need to handle the Python dependency this way. There is built-in template support for Python and other languages. See http://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html

Here's an example showing how it's done:

https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/pandas/pandas-0.20.1-intel-2017a-Python-3.6.1.eb